.circleci/config.yml

```diff
@@ -0,0 +1,18 @@
+version: 2
+jobs:
+  release:
+    docker:
+      - image: circleci/golang:1.11
+    steps:
+      - checkout
+      - run: curl -sL https://git.io/goreleaser | bash
```

Comment:
I'd prefer not to use a shortened url. Also, I think we should install a specific release. This gets you the latest version.

Reply:
Good points, done.